server/models/provider-datafields.js

```diff
@@ -0,0 +1,38 @@
+export default (sequelize, DataTypes) => {
```

Comment:
@do-kevin I don't think you need to define this model at all should be implied, can you try to remove this file, maybe just rename